README.md

```diff
@@ -21,6 +21,12 @@ install them on Ubuntu:
     sudo apt-get install mecab-ipadic-utf8 libmecab-dev
     pip3 install mecab-python3
 
+## Unicode data
+
+The tokenizers used to split non-Japanese phrases use regexes built using the
+`unicodedata` module from Python 3.4, which uses Unicode version 6.3.0.  To
```

Comment:
It occurs to me that the ftfy data is built with 3.5.  Should we do the same here?  (If so, can someone point me at a 3.5 venv I can use?)